projects/packages/forms/src/dashboard/class-dashboard.php

```diff
@@ -165,6 +182,21 @@ public function render_dashboard() {
 		<?php
 	}
 
+	/**
+	 * Render new dashboard, using the new menu parent.
+	 * TODO: once new menu is ready, deprecate the old manu item somehow (render_dashboard notice?)
+	 */
+	public function render_new_dashboard() {
+		// for now, just print a div with a message and render the old dashboard
+		?>
+		<div style="position: absolute; right: 8px; top: 8px; text-align: right;">
+			Page under construction.
+			Please use the old <a href="<?php echo esc_url( admin_url( 'edit.php?post_type=feedback' ) ); ?>">dashboard</a> until this page is ready.
+		</div>
+		<?php
+		$this->render_dashboard();
+	}
```

Comment:
For my part, I'd prefer to confirm this page works as expected in the new location. If not, address any issues. If so, remove the 'construction' notice. 

From what I can tell, the new page works fine in the new location. So I'd vote to remove the construction notice.